apps/pie-microsite/src/assets/styles/components/_tokensTable.scss

```diff
@@ -18,6 +34,36 @@ $tokensTable-item-vertical-padding: f.spacing(e);
     }
 }
 
+.c-tokensTable-example-container--spacing {
```

Comment:
not for this PR but is it worth us creating a ticket to move token specific scss into separate files, like we've done for shortcodes/ tokensTable with tokenTypes? 😄

Reply:
maybe, personally I'm fine with it all being part of the tokens table

Reply:
Mainly because I don't think it makes sense to extract modifier classes into their own files - think it would be unruly and makes sense to see the variants in the context of the surrounding styles

Reply:
okay cool, happy to keep as is - just wondered if it was getting too lengthy 👍